PedroChaparro/loomies-backend
# Test: Signup route

## Description
Includes first tests for the user controllers to test the integration workflow. 

## How to test this PR?

1. Copy the .env file inside `/controllers`.
2. Run `go test ./...` from `/api`
3. Verify that the test status is OK and that no new user has been created in the database.

## Notes 
Please, merge this PR only AFTER #35 GOES MERGED.
